### PR TITLE
build: change resize polyfill

### PR DESCRIPTION
**Pull request by wzhudev**
Ref https://github.com/ant-design/ant-design/issues/30595.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ant-design/react-slick",
-  "version": "0.28.1",
+  "version": "0.28.2",
```
Comment (afc163) [inline package.json]:
```suggestion
  "version": "0.28.1",
```

**Comment by yoyo837** [comment]
Reverted by https://github.com/ant-design/react-slick/commit/5fee151

**Comment by wzhudev** [comment]
后续的 polyfill 替换都不考虑了？

**Comment by yoyo837** [comment]
https://github.com/ant-design/ant-design/issues/30595#issuecomment-1623464984 主要看是否主流吧

**Pull request merged**